packages/core/src/components/button/_button.scss

```diff
@@ -68,6 +68,21 @@ Styleguide components.button.css
     }
   }
 
+  &.pt-loading {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
IMO flex is not over the top and I generally prefer it over `position: absolute` approaches